contracts/truefi/TrueRatingAgencyV2.sol

```diff
@@ -83,56 +83,17 @@ contract TrueRatingAgencyV2 is ITrueRatingAgencyV2, Ownable {
     uint256 public rewardMultiplier;
 
     // are submissions paused?
-    bool public submissionPauseStatus;
+    bool private DEPRECATED__submissionPauseStatus;
 
-    mapping(address => bool) public canChangeAllowance;
+    mapping(address => bool) private DEPRECATED__canChangeAllowance;
```

Comment:
Can we go further and remove those entirely?

Reply:
This is already live on mainnet: https://etherscan.io/address/0x8838f39daeb607bfb1d3cea620705099ae8fb73b#code#F20#L1

I would only remove storage variables that have not yet been deployed. Otherwise if in the future we add new storage again, then these slots could get corrupted.